src/cmd-init

```diff
@@ -180,5 +180,6 @@ mkdir -p tmp
 ostree --repo=repo init --mode=archive
 if ! has_privileges && [ ! -f cache/cache.qcow2 ]; then
     qemu-img create -f qcow2 cache/cache.qcow2 10G
-    LIBGUESTFS_BACKEND=direct virt-format --filesystem=xfs -a cache/cache.qcow2
+    # use ext4 because we hit an error on el7 when trying to use xfs
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1123221

I confirmed that this can be solves with an install of `libguestfs-xfs`

Reply:
ahh. cool. will fix that up after lunch